source.txt

```diff
@@ -497,54 +497,16 @@ Table of Contents
-    have some way to add open web app manifests [MANIFEST]. Adding a
-    manifest to the dashboard is considered equivalent to clicking
-    'accept' in the dialog of the application-first flow. Removing one
-    is considered equivalent to revoking its access token.
-
-    As an equivalent to OAuth's 'scope' parameter, a 'datastores-access'
-    field SHOULD be present in the root of such an application manifest
-    document, with entries <module> -> '{"access": "readonly"}' for
-    <level> 'r' or '{"access": "readwrite"}' for <level> 'rw', as
-    prescribed in [DATASTORE].
-
-    When the user gestures they want to use a certain application whose
-    manifest is present on the dashboard, the dashboard SHOULD redirect
-    to the application or open it in a new window. To mimic coming back
-    from the OAuth dialog, it MAY add 'access_token' and 'scope'
-    fields to the URL fragment.
-
-    Regardless of whether 'access_token' and 'scope' are specified, it
-    SHOULD add a 'remotestorage' field to the URL fragment, with a
-    value of the form <account> '@' <host>. When the application detects
-    this parameter, it SHOULD resolve the WebFinger record for <account>
-    at <host> and extract the <storage_root> and <storage_api>
-    information.
-
-    If no access_token was given, then the application SHOULD also
-    extract the <auth_endpoint> information from WebFinger, and continue
-    as per application-first bearer token issuance.
-
-    Note that whereas a remoteStorage server SHOULD offer support for
-    the application-first flow with WebFinger and OAuth, it MAY choose
-    not to support the storage-first flow, provided that users will
-    easily remember their <account> '@' <host> WebFinger address at that
-    provider. Applications SHOULD, however, support both flows, which
-    means checking the URL for a 'remotestorage' parameter, but giving
-    the user a way to specify the WebFinger address if there is none.
-
-    If a server provides an application manifest dashboard, then it
-    SHOULD merge the list of applications there with the list of
-    issued access tokens as specified by OAuth into one list. Also,
-    the interface for revoking an access token as specified by OAuth
-    SHOULD coincide with removing an application from the dashboard.
-
-    Servers MAY also provide a way to create access tokens directly from
-    their user interface. Such functionality would be aimed mainly at
-    developers, to manually copy and paste a token into a script or
-    debug tool, thus bypassing the need for an OAuth dance. Clients
-    SHOULD NOT rely on this in production.
+    To request that the application connects to the user account
+    <account> ' ' <host>, providers MAY redirect to applications with a
+    `remotestorage` field in the URL fragment, with the user account as
+    value.
+
+    The appplication MUST make sure this request is intended by the
```

Comment:
Fixed.

On Mon, May 23, 2016 at 07:47:38AM -0700, Sebastian Kippe wrote:
> > -    SHOULD merge the list of applications there with the list of
> > -    issued access tokens as specified by OAuth into one list. Also,
> > -    the interface for revoking an access token as specified by OAuth
> > -    SHOULD coincide with removing an application from the dashboard.
> > -
> > -    Servers MAY also provide a way to create access tokens directly from
> > -    their user interface. Such functionality would be aimed mainly at
> > -    developers, to manually copy and paste a token into a script or
> > -    debug tool, thus bypassing the need for an OAuth dance. Clients
> > -    SHOULD NOT rely on this in production.
> > +    To request that the application connects to the user account
> > +    <account> ' ' <host>, providers MAY redirect to applications with a
> > +    `remotestorage` field in the URL fragment, with the user account as
> > +    value.
> > +
> > +    The appplication MUST make sure this request is intended by the
> 
> Typo.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/remotestorage/spec/pull/145/files/eabaaf4670126f512fe2e33a6f464b0445362786#r64235376